lib/manageiq/graphql/types/provider.rb

```diff
@@ -0,0 +1,31 @@
+module ManageIQ
+  module GraphQL
+    module Types
+      Provider = ::GraphQL::ObjectType.define do
+        name "Provider"
+        description "A provider is a server with software to manage multiple virtual machines that reside on multiple hosts"
```

Comment:
Description is more generic than that... Some provider don't manage virtual machines at all (such as Network providers)

Reply:
He took it from http://manageiq.org/docs/get-started/concepts , which is where I've taken every description I wrote too ;) Should update both.

Reply:
I yoinked this directly from http://manageiq.org/docs/get-started/concepts - does that need updating?

Reply:
yeah that needs updating...it's probably never been updated since back when it was just cloud and infra